src/booleans.js

```diff
@@ -1,62 +1,94 @@
 function negate(a) {
   // your code here
+  return !(a > 0);
 };
 
 function both(a, b) {
   // your code here
+  return a > 0 && b > 0;
 };
 
 function either(a, b) {
   // your code here
+  return a > 0 || b > 0;
 };
 
 function none(a, b) {
   // your code here
+  return !(a > 0 || b > 0);
 };
 
 function one(a, b) {
   // your code here
+  if (a > 0 && b > 0) {
+    return false;
+  }
+  return a > 0 || b > 0;
 };
+  
+
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
   // your code here
+  return Boolean(a === b);
 };
 
 function isGreaterThan(a, b) {
   // your code here
+  return Boolean(a > b);
 };
 
 function isLessThanOrEqualTo(a, b) {
   // your code here
+  return Boolean(a <= b);
 };
 
 function isOdd(a) {
   // your code here
+  return Boolean(a % 2);
 };
 
 function isEven(a) {
   // your code here
+  return Boolean(a % 2 == 0);
 };
 
 function isSquare(a) {
   // your code here
+  if (a < 0) return false;
+  return Boolean(Math.sqrt(a) % 1 === 0);
 };
 
 function startsWith(char, string) {
   // your code here
+  if (string.startsWith(char)) return true;
+  return false;
```

Comment:
Recursion! This is some thing you rarely see in the wild, and it has very specific use cases so is generally not a good idea to use. Instead, using a simpler comparison would be the way to go:
```
  return char === string[0];
```